### PR TITLE
closes #63 release prep

### LINKED ISSUE #63: General Issue: 1.0 release
### Background Information and pre-release tasks

- Release after admiral releases on week of 3rd June
- update description file and changelog
- change icon in chrome tab to package hex
- add @sadchla-codes as developer

### Definition of Done

- devtools::release()

### LINKED ISSUE #71: General Issue: Remove non-ASCII characters in the package ahead of any release
### Background Information

Non-ASCII characters cause NOTES to appear when running pre-release checks. We should remove any from the package to avoid any issues.

### Definition of Done

- Run the following in the searchbar on github:
 `repo:pharmaverse/pharmaverseadam /[^\x00-\x7F]+/`
- Remove anything flagged.

### PR DESCRIPTION
**Pull request by sadchla-codes**
Thank you for your Pull Request! We have developed this task checklist from the [Development Process Guide](https://pharmaverse.github.io/admiral/CONTRIBUTING.html#detailed-development-process) to help with the final steps of the process. Completing the below tasks helps to ensure our reviewers can maximize their time on your code as well as making sure the admiral codebase remains robust and consistent.   

Please check off each taskbox as an acknowledgment that you completed the task or check off that it is not relevant to your Pull Request. This checklist is part of the Github Action workflows and the Pull Request will not be merged into the `devel` branch until you have checked off each task.

- [x] Place Closes #<insert_issue_number> into the beginning of your Pull Request Title (Use Edit button in top-right if you need to update)
- [x] Code is formatted according to the [tidyverse style guide](https://style.tidyverse.org/). Run `styler::style_file()` to style R and Rmd files
- [x] Updated relevant unit tests or have written new unit tests, which should consider realistic data scenarios and edge cases, e.g. empty datasets, errors, boundary cases etc. - See [Unit Test Guide](https://pharmaverse.github.io/admiraldev/articles/unit_test_guidance.html#tests-should-be-robust-to-cover-realistic-data-scenarios)
- [x] If you removed/replaced any function and/or function parameters, did you fully follow the [deprecation guidance](https://pharmaverse.github.io/admiraldev/articles/programming_strategy.html#deprecation)?
- [x] Update to all relevant roxygen headers and examples, including keywords and families. Refer to the [categorization of functions](https://pharmaverse.github.io/admiraldev/articles/programming_strategy.html#categorization-of-functions) to tag appropriate keyword/family.
- [x] Run `devtools::document()` so all `.Rd` files in the `man` folder and the `NAMESPACE` file in the project root are updated appropriately
- [x] Address any updates needed for vignettes and/or templates
- [x] Update `NEWS.md` if the changes pertain to a user-facing function (i.e. it has an `@export` tag) or documentation aimed at users (rather than developers)
- [x] Build admiral site `pkgdown::build_site()` and check that all affected examples are displayed correctly and that all new functions occur on the "[Reference](https://pharmaverse.github.io/admiral/reference/index.html)" page. 
- [x] Address or fix all lintr warnings and errors - `lintr::lint_package()`
- [x] Run `R CMD check` locally and address all errors and warnings - `devtools::check()`
- [x] Link the issue in the Development Section on the right hand side.
- [x] Address all merge conflicts and resolve appropriately 
- [x] Pat yourself on the back for a job well done! Much love to your accomplishment!

### CODE REVIEW

File: NEWS.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+# pharmaverseadam 1.0.0
+
+## New Features
```
Comment (manciniedoardo) [inline NEWS.md:3]: Could we please update here so that it's not completely empty under 1.0.0, otherwise it begs the question as to why we are doing a release at all! Maybe you could replace the "New features" with some other title and write something along the lines of "refreshed dataset with ADaM datasets created by admiral 1.1.1, admiralophtha 1.1, etc..."

**Review by manciniedoardo — CHANGES_REQUESTED** [review]
Just one note :)

File: NEWS.md [hunk 2]
```diff
@@ -1,3 +1,10 @@
+# pharmaverseadam 1.0.0
+
+## Documentation
+  - The first stable version of `pharmaverseadam`!
+  - Dataset labels are updated to align with ADaM IG (#64) 
+  - Refreshed dataset with ADaM datasets created by admiral 1.1.1, admiralophtha 1.1 (#58)
```
Comment (manciniedoardo) [inline NEWS.md:6]:
```suggestion
  - Refreshed `{pharmaverseadam}` datasets using {admiral} 1.1.1}, {admiralophtha} 1.1.0, {admiralonco} 1.1.0, `{admiralvaccine}` 0.2.0. (#58)
```

File: NEWS.md [hunk 3]
```diff
@@ -1,3 +1,10 @@
+# pharmaverseadam 1.0.0
+
+## Documentation
+  - The first stable version of `pharmaverseadam`!
```
Comment (manciniedoardo) [inline NEWS.md:4]:
```suggestion
  - The first stable version of `{pharmaverseadam}`!
```

**Review by manciniedoardo — CHANGES_REQUESTED** [review]
Minor updates

**Review by manciniedoardo — APPROVED** [review]
